nixos/modules/services/misc/sr-ht/builds.nix

```diff
@@ -0,0 +1,183 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.srht;
+  bcfg = cfg.builds;
+in {
+  options = {
+    services.srht.builds = {
+      redis = mkOption {
+        type = types.str;
+        default = "redis://localhost:6379/0";
+        description = ''
+          The redis connection used for the Celery worker.
+        '';
+      };
+
+      worker.user = mkOption {
+        type = types.nullOr types.str;
+        default = "runner";
+        description = ''
+          builds.sr.ht runner user.
+        '';
+      };
+
+      worker.name = mkOption {
+        type = types.str;
+        default = "runner.sr.ht.local";
+        description = ''
+          Name of this build runner.
+        '';
+      };
+
+      # TODO: This needs a better default...
+      # Instead of my random path...
+      worker.buildLogs = mkOption {
+        type = types.path;
+        default = "/var/srht/logs";
+        description = ''
+          Path to the build logs.
+        '';
+      };
+
+      worker.images = mkOption {
+        type = types.path;
+        default = "${bcfg.package}/lib/images";
+        description = ''
+          Path to the build images.
+        '';
+      };
+
+      worker.controlCommand = mkOption {
+        type = types.path;
+        default = "${bcfg.package}/lib/images/control";
+        description = ''
+          In production you should NOT put the build user in the docker group. Instead,
+          make a scratch user who is and write a sudoers or doas.conf file that allows
+          them to execute just the control command, then update this config option. For
+          example:
+
+          doas -u docker /var/lib/images/control
+
+          Assuming doas.conf looks something like this:
+
+          permit nopass builds as docker cmd /var/lib/images/control
+
+          For more information about the security model of builds.sr.ht, visit the wiki:
+
+          https://man.sr.ht/builds.sr.ht/security
```

Comment:
This URL doesn't exist?

Reply:
Looks like the example config is outdated in some parts, looks like it's now in with the installation instructions? https://man.sr.ht/builds.sr.ht/installation.md#security-model

Reply:
Perhaps this was meant to be https://man.sr.ht/builds.sr.ht/installation.md#security-model ?